.azure-devops/explore.config.yml

```diff
@@ -0,0 +1,44 @@
+resources:
+- repo: self
+  clean: true
+
+pool:
+  name: Team Pool
+  demands: npm
+
+steps:
+- task: NodeTool@0
+  displayName: 'Use Node 10.14.x'
+  inputs:
+    versionSpec: 10.14.x
+
+- task: Npm@1
+  displayName: 'Install Lerna'
+  inputs:
+    command: custom
+    verbose: false
+    customCommand: 'i lerna -g'
+
+- task: Npm@1
+  displayName: 'Install Package Dependencies'
+  inputs:
+    verbose: false
+
+- script: 'lerna bootstrap'
+  displayName: 'Lerna Bootstrap'
+
+- task: Npm@1
+  displayName: 'Build Component Explorer Website'
```

Comment:
Should we specify to say "Build Microsoft Component Explorer Website"?

Reply:
While it only has Microsoft components today, I believe it more likely to have buckets of different component libraries going forward.  For example, maybe we want to add a few of Fabric's components in here since they are already OSS. Similar to how we have reshaped the internal component explorer.